In some cases only one bitmap is found when pbm2bm builds the list
of free bitmaps.  Unfortunately the next bitmap pointer doesn't
get initialized in that case causing a crash.  This patch ensures
that the next free bitmap pointer is always initialized.

-- John Wehle

--- pbm2cl.c	2003-01-29 07:53:04.000000000 -0500
+++ pbm2cl.c	2008-06-26 15:41:18.000000000 -0400
@@ -1048,9 +1048,9 @@
         for (i=j=0, la=-1; i<abs; ++i) {
             if (ab[i].x < 0) {
+                ab[i].y = -1;
                 if (la < 0)
                     la = j = i;
                 else {
                     ab[j].y = i;
-                    ab[i].y = -1;
                     j = i;
                 }
